modules/runner/src/main/java/org/apache/ignite/internal/storage/DistributedConfigurationStorage.java

```diff
@@ -83,7 +84,7 @@
     private volatile ConfigurationStorageListener lsnr;
 
     /** Storage version. It stores actual meta storage revision, that is applied to configuration manager. */
-    private final AtomicLong ver = new AtomicLong(0L);
+    private final AtomicLong changeId = new AtomicLong(0L);
```

Comment:
This is not a storage version anymore, we should reflect that in the comment

Reply:
Well, technically that is still revision of the local metastorage at the moment of last configuration update (including node start). Maybe I should clarify that in the comment?

Reply:
I updated the comment